config/routes.rb

```diff
@@ -302,10 +302,10 @@ def draw(routes_name)
       end
     end
 
-    resources :project_folders, only: [] do
-      get '/', to: 'projects#index'
+    resources :project_folders, only: %i(new create) do
       get 'cards', to: 'projects#cards'
     end
+    get 'project_folders/:project_folder_id', to: 'projects#index', as: :project_folder
```

Comment:
Why is it moved here?

Reply:
Because before non of the other routes were working, because they were always routed to `projects#index`